geth/exceptions.py

```diff
@@ -2,24 +2,51 @@
     annotations,
 )
 
+import codecs
 import textwrap
-
-from .utils.encoding import (
-    force_text,
+from typing import (
+    Any,
 )
 
 
-def force_text_maybe(value: str | bytes | bytearray | None) -> str | None:
-    if value is not None:
-        return force_text(value, "utf8")
-    return None
+def force_text_maybe(value: bytes | bytearray | str | None) -> str | None:
```

Comment:
This is written out in full here due to circular import: `encoding.py` imports `PyGethTypeError` from this file, and this file was importing `force_text` from `encoding.py`